README.md

```diff
@@ -30,6 +30,8 @@ Configure using an API token from Circle
 ```ruby
 CircleCi.configure do |config|
   config.token = 'my-token'
+  config.host = 'https://circleci.com/api/v1'
+  config.port = '80'
```

Comment:
Port isn't being used at the moment so no point in making it configurable for now so lets revert that please.